packages/extension-ui/src/Popup/Export.tsx

```diff
@@ -45,16 +59,20 @@ function Export({
       .then(({ exportedJson }) => {
         const blob = new Blob([JSON.stringify(exportedJson)], { type: 'application/json; charset=utf-8' });
 
-        saveAs(blob, `${address}.json`);
-
-        onAction('/');
+        saveAs(blob, `AlephZeroSigner_${address}.json`);
+        show(t<string>('Export successful'), 'success');
+        onAction('..');
       })
       .catch((error: Error) => {
         console.error(error);
         setError(error.message);
         setIsBusy(false);
       });
-  }, [address, onAction, pass]);
+  }, [address, onAction, pass, show, t]);
+
+  const _handleInputTypeChange = useCallback(() => {
+    setIsPasswordVisible(!isPasswordVisible);
+  }, [isPasswordVisible]);
```

Comment:
Probably mostly the same, but less prone to refactor errors and produces new reference less often (just once, to be exact) if we're playing by those _`useCallbacks` everywhere_ rules 😃 :
```suggestion
  const _handleInputTypeChange = useCallback(
    () => setIsPasswordVisible(isPasswordVisible => !isPasswordVisible),
    [],
  );
```